### DISCUSSION TITLE
怎么支持raw格式的照片

### DISCUSSION THREAD

**Discussion by Dnyo666** [lsky-org/lsky-pro#441] [问答]
有些照片是raw格式的CR3的照片，想问下有没有办法能支持这种格式

**Comment by 0xxb** [comment]
暂时无法支持这种格式，即使强行上传了也无法预览、生成缩略图、加水印等。

**Comment by Dnyo666** [comment]
希望可以在以后的更新中加入raw格式




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Wisp ***@***.***&gt;; 
发送时间: 2022年6月6日(星期一) 上午8:22
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [lsky-org/lsky-pro] 怎么支持raw格式的照片 (Discussion #441)






暂时无法支持这种格式，即使强行上传了也无法预览、生成缩略图、加水印等。

—
直接回复此邮件，在 GitHub 上查看或取消订阅.
您收到这个消息是因为您编写了这个线程。留言 ID ***@***.***与&gt;
